crates/nu-command/src/conversions/into/cell_path.rs

```diff
@@ -172,6 +180,7 @@ fn record_to_path_member(
 
 fn value_to_cell_path(value: &Value, span: Span) -> Result<Value, ShellError> {
     match value {
+        Value::CellPath { .. } => Ok(value.clone()),
```

Comment:
I know you didn't originally make this, but could you remove the borrow from this function so we don't have to clone the cell path?